src/App.vue

```diff
@@ -240,8 +240,7 @@ export default {
         }
       } finally {
         this.shown = true
-
-        if (!this.$route.params.tenant) {
+        if (this.isCloud && !this.$route.name && !this.$route.params.tenant) {
```

Comment:
Hm I don't think this is correct; this shouldn't be restricted to Cloud (tenants exist in Server now so we still want the slug), and the Dashboard is a named route, aliased to root:
```js
// router.js
 {
    name: 'dashboard',
    alias: '/',
    path: '/:tenant?',
    component: () =>
      import(
        /* webpackChunkName: "dashboard" */ '@/pages/Dashboard/Dashboard.vue'
      ),
    beforeEnter: multiguard([authNavGuard, tenantNavGuard])
  }

```